invenio_rdm_records/records/mappings/v6/rdmrecords/records/record-v1.0.0.json

```diff
@@ -56,6 +56,9 @@
             }
           }
         },
+        "has_draft": {
```

Comment:
Minor: Should this one also be added to the drafts mapping to keep the two mappings identical......if we dont' add data it's just ignored, but I'd prefer not having two different mappings for drafts and records.